crypto/rand/rand_lcl.h

```diff
@@ -94,21 +94,34 @@ struct rand_drbg_st {
     int nid; /* the underlying algorithm */
     int fork_count;
     unsigned short flags; /* various external flags */
-    char filled;
     char secure;
     /*
      * This is a fixed-size buffer, but we malloc to make it a little
      * harder to find; a classic security/performance trade-off.
      */
     int size;
-    unsigned char *randomness;
 
-    /* These parameters are setup by the per-type "init" function. */
+    /* The following parameters are setup by the per-type "init" function.
```

Comment:
Here is the added comment:

Reply:
oops, the "/*" goes on a line by itself.